src/constants/index.js

```diff
@@ -2,5 +2,6 @@ module.exports = {
   AFRAME_INJECTED: 'aframe-injected',
   DEFAULT_CAMERA_HEIGHT: 1.6,
   DEFAULT_HANDEDNESS: 'right',
-  keyboardevent: require('./keyboardevent')
+  keyboardevent: require('./keyboardevent'),
+  AFRAME_CDN_ROOT: window.AFRAME_CDN_ROOT || 'https://cdn.aframe.io/'
```

Comment:
This list seems to be sorted, you should put the new constant first, before AFRAME_INJECTED.

Reply:
Yes, well spotted.  Now alphabetized.